### PR TITLE
Elections update for expired events

### PR DESCRIPTION
**Pull request by jeffrummel**

### CODE REVIEW

File: layouts/shortcodes/sidebar-link.html [hunk 1]
```diff
@@ -37,6 +38,18 @@
         >Add to Calendar</span
       >
     </button>
+  {{ else if $dateExpired }}
```
Comment (carlmjohnson) [inline layouts/shortcodes/sidebar-link.html:41]: Can't we just test if today is after the date?

Comment (carlmjohnson) [reply layouts/shortcodes/sidebar-link.html:41]: `time "2023-08-01"` turns a date string into time object. Then you can compare it with the `now` time object. Something like

```
{{ $dateObj := time $date }}
{{ if now.After $dateObj }}
 ```

https://pkg.go.dev/time#Time.After documents the methods on a time object.

Comment (jeffrummel) [reply layouts/shortcodes/sidebar-link.html:41]: hugo can't seem to parse the date from the shortcode. 

`{{ $date := "2023-08-01" }}` works, but `{{ $date := .Get "date" }}` does not.

Comment (carlmjohnson) [reply layouts/shortcodes/sidebar-link.html:41]: I did a quick test and I ran into the problem that the build fails if `time $date` is "", but I think it should be okay if it's behind an `if $date` statement.

Comment (carlmjohnson) [reply layouts/shortcodes/sidebar-link.html:41]: Opened a sub-PR.

**Comment by carlmjohnson** [comment]
I'm going to go ahead and merge it just so it's done before the weekend.

**Pull request merged**